### ISSUE TITLE
Paket friendly Fabulous package

### ISSUE THREAD

**Issue by sergey-tihon** [fsprojects/Fabulous#328]
Not sure if it is actually `Fabulous` issues (just want to track is somewhere), but I think that would be nice to allow to manage `Fabulous` package using `Paket` easier.

I did migrate my test application (generated from Fabulous template to Paket) -  https://github.com/sergey-tihon/MinMe/commit/fb67ee59d4e55ac191a192658849d3a89698fdb7
It works, but require additional manual changes in project file after `paket.exe install` or `update`.

`App.macOS.fsproj` file is in old format and Paket insert all dependencies in proj file. In my case, project targets
```xml
<TargetFrameworkIdentifier>Xamarin.Mac</TargetFrameworkIdentifier>
```

Paket generate following XML during install
```xml
  <Choose>
    <When Condition="$(TargetFrameworkIdentifier) == '.NETStandard' And $(TargetFrameworkVersion) == 'v2.0'">
      <ItemGroup>
        <Reference Include="Fabulous.Core">
          <HintPath>..\..\packages\Fabulous.Core\lib\netstandard2.0\Fabulous.Core.dll</HintPath>
          <Private>True</Private>
          <Paket>True</Paket>
        </Reference>
      </ItemGroup>
    </When>
  </Choose>
```

Then MSBuild cannot compile project for `Xamarin.Mac` because it does not see reference to `Fabulous.Core.dll`.

### Workaround

Manually update condition in `fsproj` file for `Fabulous.Core.dll` and `Fabulous.CustomControls.dll`.

From 
```xml
<When Condition="$(TargetFrameworkIdentifier) == '.NETStandard' And $(TargetFrameworkVersion) == 'v2.0'">
```

To
```xml
<When Condition="($(TargetFrameworkIdentifier) == '.NETStandard' And $(TargetFrameworkVersion) == 'v2.0') 
Or ($(TargetFrameworkIdentifier) == 'Xamarin.Mac')">
```

**Comment by TimLariviere** [comment]
Last I checked, Paket is not really maintained up to date for Xamarin.
A few months ago, I had to submit a PR to make it work for Xamarin.Android and Xamarin.iOS. (Paket thought Xamarin.Android/iOS didn't supported .NET Standard 2.0)

Seems to be a similar issue.

**Comment by sergey-tihon** [comment]
Should be fixes starting from Paket `5.203.1`
